src/dev/src/app/utils/demo-page.component.ts

```diff
@@ -0,0 +1,62 @@
+import { AfterContentInit, Component, ContentChildren, Input, QueryList } from '@angular/core';
+import { DemoComponent } from './demo.component';
+
+/**
+ * Class:  Demo-page.component.ts
+ *
+ * Description:
+ * ---------------------------------
+ * This component is building a Demo Page, it takes a pageTitle as parameter
+ * and contains a list of Demo Components.
+ *
+ * Example:
+ * ---------------------------------
+ * <demo-page pageTitle="Widget Component">
+ *   <demo-component [componentData]="examples[0]"></demo-component>
+ *   <demo-component [componentData]="examples[1]"></demo-component>
+ * </demo-page>
+ *
+ */
+@Component({
+  selector: 'demo-page',
+  template: `<h2 class="mt-2 mb-2">{{pageTitle}}</h2>
+  <p class="mt-4">Filters :
+    <button class="btn btn-sm btn-info" *ngIf="canDisable" (click)="setDisable()">Toggle Disabled ({{disabled ? 'true' : 'false'}})</button>
+    <button class="btn btn-sm btn-info ml-2" (click)="toggleAllCodes()">Toggle all code</button>
+    <button class="btn btn-sm btn-info ml-2" (click)="toggleAllResults()">Toggle all results</button>
```

Comment:
You need to display the toggleResults only if the is models.

Reply:
Done